lib/avtp_pipeline/openavb_common/avb.c

```diff
@@ -69,12 +69,12 @@ int pci_connect(device_t *igb_dev)
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, igb_dev);
 		if (err) {
-			printf("attach failed! (%s)\n", strerror(err)); /*errno or err ??*/
+			printf("attach failed! (%s)\n", strerror(err));
 			continue;
 		}
 		err = igb_attach_tx(igb_dev);
 		if (err) {
-			printf("attach_tx failed! (%s)\n", strerror(err)); /*errno or err ??*/
+			printf("attach_tx failed! (%s)\n", strerror(err));
```

Comment:
I like these changes, but I'm left with the question about what it means to call strerror() with an error returned from a user library. Seems to me there should be a igb_strerror() function since the igb library is the entity that understands what it's returned error codes actually mean.